relayer/chainreader/reader/chainreader.go

```diff
@@ -195,6 +195,11 @@ func (s *suiChainReader) GetLatestValue(ctx context.Context, readIdentifier stri
 			structResult[mapKey] = results[i]
 		}
 
+		// Apply result field renames if configured
+		if functionConfig.ResultFieldRenames != nil {
+			structResult = applyResultFieldRenames(structResult, functionConfig.ResultFieldRenames).(map[string]any)
```

Comment:
same comment as (https://github.com/smartcontractkit/chainlink-sui/pull/42) 

why not keep this outside the if statement, so we always apply renames if `ResultFieldRenames` is present, we won't use it in Core and only be used in o11y, so we should be safe right?